scrapers/czechcasting.yml

```diff
@@ -0,0 +1,23 @@
+name: czechav
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - czechcasting.com/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    common:
+      $studio: //*[@id="logo"]/a/img/@alt
+    scene:
+      Title: /html/head/title/text()
+      Details: //*[@id="girl-info"]/div/div/div[2]/p/text()
+      Tags:
+        Name: //*[@id="girl-info"]/div/div/ul[@class='tags']/li/a[@class='tag']/text()
```

Comment:
```yaml
        Name: //ul[@class='tags']/li/a[@class='tag']/text()
```
is enough i think

Reply:
Ah that makes sense. not sure why i was being overly specific